packages/core-mobile/app/screens/send/utils/evm/buildEVMSendTx.ts

```diff
@@ -0,0 +1,149 @@
+import ERC20 from '@openzeppelin/contracts/build/contracts/ERC20.json'
+import ERC721 from '@openzeppelin/contracts/build/contracts/ERC721.json'
+import ERC1155 from '@openzeppelin/contracts/build/contracts/ERC1155.json'
+import { JsonRpcBatchInternal } from '@avalabs/core-wallets-sdk'
+import { Contract, TransactionRequest } from 'ethers'
+import {
+  NftTokenWithBalance,
+  TokenType,
+  TokenWithBalanceERC20
+} from '@avalabs/vm-module-types'
+import { EVMSendOptions } from '../types'
+
+export const buildErc20Tx = async ({
+  from,
+  provider,
+  toAddress,
+  amount,
+  token
+}: {
+  from: string
+  provider: JsonRpcBatchInternal
+  toAddress: string
+  amount: bigint
+  token: TokenWithBalanceERC20
+}): Promise<TransactionRequest> => {
+  const contract = new Contract(token.address || '', ERC20.abi, provider)
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const populatedTransaction = await contract.transfer!.populateTransaction(
+    toAddress,
```

Comment:
I think you can use the contracts from `app/contracts/openzeppelin/` here. it has typescript support.

Reply:
fixed.